cmd/data-node/commands/load_all_available_history.go

```diff
@@ -0,0 +1,70 @@
+package commands
```

Comment:
I think this whole file can be removed now that its been moved into `cmd/data-node/commands/history/load.go`?